python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1735,6 +1770,10 @@ def computeAltMaskList(self, tempExpRefList, maskSpanSets):
             for noDataRegion in noData:
                 noDataRegion.clippedTo(mask.getBBox()).setMask(mask, 2**noDataValue)
             altMaskList.append(mask)
+            if lsstDebug.Info(__name__).saveAltMask:
+                filename = 'altMask-%s.fits' % ('-'.join([str(v) for v in warpRef.dataId.values()]))
```

Comment:
Similar approach as above, but won't produce the same suffix. Suggest you consolidate the code to produce the suffix, and then they'll match.